### ISSUE TITLE
Comparison to async-mpd?

### ISSUE THREAD

**Issue by blakehawkins** [elomatreb/mpd_client#6]
👋 I see this is an active project. I'm looking for an mpd backend and considered this versus https://lib.rs/crates/async-mpd .

One thing I noticed is that your lib doesn't use any unsafe code (👍)

Is there any other reason that you're not just using the existing project?

Thanks in advance!

**Comment by elomatreb** [comment]
Back when I started working on this while working on a long-lived side project for an MPD client that fits my needs, the other project didn't exist :)

I extracted the first versions of this from application code I had written, and while I abandoned the original attempt at that project, I am still using it in the "next" version I am working on when I have the time. A major motivation has also been just to try stuff, I started with no experience with Rust and I used this project to try writing parsers and async Rust in general.

So far I'm the only user of this library I know of and its design is shaped by the needs I had while working on an MPD client, but if you have any specific suggestions I am open to hearing them.
